esp/services/ws_dfu/ws_dfuXRefService.cpp

```diff
@@ -223,9 +223,17 @@ void CWsDfuXRefEx::readLostFileQueryResult(IEspContext &context, StringBuffer& b
         if (!fileName || !*fileName)
             continue;
 
-        Owned<IDistributedFile> df = queryDistributedFileDirectory().lookup(fileName, userdesc, false, false, true); // lock super-owners
-        if(df)
-            item.addPropInt64("Size", queryDistributedFileSystem().getSize(df));
+        try
+        {
+            Owned<IDistributedFile> df = queryDistributedFileDirectory().lookup(fileName, userdesc, false, false, false, NULL, 0);
+            if(df)
+                item.addPropInt64("Size", queryDistributedFileSystem().getSize(df));
+        }
+        catch(IException* e)
+        {
+            item.addProp("Status", "Warning: this file may be locked now. It can't be recovered as locked.");
```

Comment:
Probably worth logging the error as well, in case it's something other than a lock timeout..